apps/spkac.c

```diff
@@ -138,8 +138,15 @@ int spkac_main(int argc, char **argv)
         if (challenge != NULL)
             ASN1_STRING_set(spki->spkac->challenge,
                             challenge, (int)strlen(challenge));
-        NETSCAPE_SPKI_set_pubkey(spki, pkey);
-        NETSCAPE_SPKI_sign(spki, pkey, EVP_md5());
+        if (!NETSCAPE_SPKI_set_pubkey(spki, pkey)) {
```

Comment:
Shouldn't be ```NETSCAPE_SPKI_set_pubkey(spki, pkey) == 0```?

Reply:
No - the return is a boolean (only ever 0 or 1), so treating as a boolean with `!` is fine

Reply:
Got it, thanks!